frontend/src/components/instaBar.js

```diff
@@ -1,13 +1,14 @@
 import * as React from "react";
 import { InstaIcon } from "./svgs";
 
+// could make entire bar clickable to link to ig page?
```

Comment:
this is a phil question.  message him in teams about this.

Reply:
ive messaged him, there is a separate issue for this. https://github.com/hmcc-global/hmccaa-web/issues/128